src/map/mob.cpp

```diff
@@ -4311,17 +4311,18 @@ static bool mob_readdb_mobavail(char* str[], int columns, int current)
 	mob_db_data[mob_id]->vd.class_ = sprite_id;
 
 	//Player sprites
-	if(pcdb_checkid(sprite_id) && columns==12) {
+	if(pcdb_checkid(sprite_id) && columns==13) {
```

Comment:
Might be better to change this to > 12 and add column 13 as optional inside with a default value.